packages/base-zone/src/prepare-revocable.js

```diff
@@ -77,13 +77,24 @@ import { wrapperMethods } from './prepare-attenuator.js';
  *   The method names of the underlying exo class that should be represented
  *   by transparently-forwarding methods of the revocable caretaker.
  * @param {RevocableKitOptions<U>} [options]
- * @returns {RevocableMakerKit<U>}
+ *
+ * @param {(amp: Amplify<any>) => void} [receiveAmplifier]
+ *   This parameter is implemented by the `FarClassOptions['receiveAmplifier']`
+ *   option to `exoClassKit`.
+ *
+ *   If provided by the caller, it will be called back exactly once to
+ *   expose the `Amplify` function which, when called on any of a Kit
+ *   instance's facets, will return the complete `facets` object.
+ *
+ *   `receiveAmplifier` is specified by `prepareRevocableMakerKit` to allow
+ *   `revoke(myKit.revocable)` to obtain and call `myKit.revoker.revoke()`.
  */
-export const prepareRevocableMakerKit = (
+export const prepareInternalRevocableMakerKit = (
   zone,
   uKindName,
   uMethodNames,
   options = {},
+  receiveAmplifier = undefined,
```

Comment:
What about

```suggestion
  options = {
    receiveAmplifier = undefined,
  },
```

?

Reply:
I'd rather not mix together the `RevocableKitOptions` and `FarOptions`.  Maybe nest the `FarOptions` at a different level instead?
```suggestion
  { exoClassOptions = {}, ...revocableKitOptions } = {},
```